src/api/PCOL/useAddPCOL.d.ts

```diff
@@ -1,5 +1,6 @@
 import "@pnp/sp/content-types";
 export declare const useAddPCOL: (subSite: string) => import("@tanstack/react-query").UseMutationResult<any, Error, {
+    Disclaimers: string[];
```

Comment:
Can't comment on the useAddPCOL.ts file directly since it wasn't changed. But it needs to set first stage as "Draft" not "New" so button to move forward becomes enabled.